packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -135,6 +135,23 @@ class TerraService {
       }
     }
   }
+
+  /**
+   * Gets executed once before all workers are shut down.
+   * Uploads the reference screenshots to the remote repository if this build was triggered from a PR merge.
+   * @param {Object} config wdio configuration object
+   */
+  // async onComplete(_, config) {
+  //   try {
+  //     if (this.serviceOptions.useRemoteReferenceScreenshots /* && This is from a PR merge  */ ) {
```

Comment:
@BenBoersma I added this change to upload the screenshots after completion of the wdio test only for a PR merge build. This block of code is currently commented out because it has a dependency on the `Cause` string from the change you're adding to indicate whether or not this is for a PR merge build.

Reply:
Roger, I have started my work for terra-functional-testing using a branch of this branch.